src/envo/envo-edit.owl

```diff
@@ -37280,6 +37361,38 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000140> "f
 EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_02000140> ObjectUnionOf(<http://purl.obolibrary.org/obo/ENVO_01000797> <http://purl.obolibrary.org/obo/ENVO_01000798> <http://purl.obolibrary.org/obo/ENVO_01000815>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000140> <http://purl.obolibrary.org/obo/ENVO_00010483>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000141> (refinery)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000141> "a human construction which is used to refine certain materials or converting raw material into products of value.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000141> <https://orcid.org/0000-0002-6601-2165>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000141> "refinery"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000141> <http://purl.obolibrary.org/obo/ENVO_00000070>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000142> (petroleum oil refinery)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Oil_refinery") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000142> "A refinery which is used for processing petroleum oil. Petroleum processing is an industrial process  in which crude oil is transformed and refined into useful products such as petroleum naphtha, gasoline, diesel fuel, asphalt base, heating oil, kerosene, liquefied petroleum gas, jet fuel and fuel oils.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000142> <https://orcid.org/0000-0002-6601-2165>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_02000142> "oil refinery")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000142> "petroleum oil refinery"@en)
+EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_02000142> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_02000141> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/ENVO_02000145> <http://purl.obolibrary.org/obo/ENVO_00002984>)))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000142> <http://purl.obolibrary.org/obo/ENVO_02000141>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000143> (sugar refinery)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Sugar_refinery") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000143> "a refinery which is used for processing raw sugar. Raw sugar is processed from cane or beets into white refined sugar.")
```

Comment:
Capitalisation and gloss extraction as above. Also, there's something off about these definitions - one doesn't "use" the whole refinery for this, and some which are inactive are still refineries. 

Perhaps a pattern like:

>A refinery which is capable of processing raw sugar.

Reply:
done

Reply:
Gloss extraction to comment on:
>Raw sugar is processed from cane or beets into white refined sugar.